play_motion:
  motions:
    look_down:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [0.0, -0.2]
        time_from_start: 0.0
    # look_left:
    #   joints: [head_1_joint, head_2_joint]
    #   points:
    #   - positions: [0.5, -0.25]
    #     time_from_start: 0.0 
    look_down_left:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [0.5, -0.55]
        time_from_start: 0.0
    # look_right:
    #   joints: [head_1_joint, head_2_joint]
    #   points:
    #   - positions: [-0.5, -0.25]
    #     time_from_start: 0.0 
    look_down_right:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [-0.5, -0.55]
        time_from_start: 0.0
    
    look_center:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [0.0, 0.0]
        time_from_start: 0.0

    look_left:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [0.5, 0.0]
        time_from_start: 0.0

    look_right:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [-0.5, 0.0]
        time_from_start: 0.0